js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/PolicyEvaluationTable.tsx

```diff
@@ -163,29 +217,29 @@ const PartitionedPolicyEvaluationTable = ({
                       description={description}
                       status={AssetConditionEvaluationStatus.TRUE}
                       subset={trueSubset}
-                      width={Math.ceil((numTrue / total) * FULL_SEGMENTS_WIDTH)}
+                      selectPartition={selectPartition}
                     />
                   ) : null}
                   {numFalse > 0 ? (
                     <PartitionSegmentWithPopover
                       status={AssetConditionEvaluationStatus.FALSE}
                       description={description}
                       subset={falseSubset}
-                      width={Math.ceil((numFalse / total) * FULL_SEGMENTS_WIDTH)}
+                      selectPartition={selectPartition}
                     />
                   ) : null}
                   {numSkipped > 0 ? (
                     <PartitionSegmentWithPopover
                       status={AssetConditionEvaluationStatus.SKIPPED}
                       description={description}
                       subset={candidateSubset}
-                      width={Math.ceil((numSkipped / total) * FULL_SEGMENTS_WIDTH)}
+                      selectPartition={selectPartition}
                     />
                   ) : null}
                 </Box>
               </td>
               <td>
-                <TimeElapsed startUnix={startTimestamp} endUnix={endTimestamp} />
+                <TimeElapsed startUnix={startTimestamp} endUnix={endTimestamp} msec />
```

Comment:
Looks like this was merged as `showMsec`?

Reply:
yeah I need to rebase